### PR TITLE
DOC: Add Trove classifiers for supported Python versions

### PR DESCRIPTION
**Pull request by moreati**
Adds missing `Programming Language :: Python :: 3` to bento.info
Adds new `Programming Language :: Python :: {2..,3..,CPython..}` to both

These versions are correct for the 1.10.x and 1.11.x releases.
They reflect the commit b06dbc15d9db0e224d338c84ac98a925a7945d4c

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -38,7 +38,15 @@
 License :: OSI Approved
 Programming Language :: C
 Programming Language :: Python
+Programming Language :: Python :: 2
+Programming Language :: Python :: 2.6
+Programming Language :: Python :: 2.7
 Programming Language :: Python :: 3
+Programming Language :: Python :: 3.2
+Programming Language :: Python :: 3.3
```
Comment (seberg) [inline setup.py]: We do not actually support 3.2 I think? Not sure about the rest ;).

Comment (moreati) [inline setup.py]: According to [1.11.0-notes.rst](https://github.com/numpy/numpy/blob/9cd868a00cc488478ac4f07ee0b5452a38cb3cc0/doc/release/1.11.0-notes.rst), [1.10.0-notes.rst](https://github.com/numpy/numpy/blob/9cd868a00cc488478ac4f07ee0b5452a38cb3cc0/doc/release/1.10.0-notes.rst), and [setup.py](https://github.com/numpy/numpy/blob/9cd868a00cc488478ac4f07ee0b5452a38cb3cc0/setup.py#L25) Python 3.2 is supported.

Comment (seberg) [inline setup.py]: Ops, you are right, we do test 3.2 on travis, something in the back of my head thought we stopped.

**Comment by rgommers** [comment]
LGTM, merged. Thanks @moreati

**Comment by charris** [comment]
Looks like another backport ;)

**Pull request merged**